src/edu/csus/ecs/pc2/ui/SubmitSampleRunsPane.java

```diff
@@ -0,0 +1,880 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.ui;
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.FlowLayout;
+import java.awt.Font;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.logging.Level;
+
+import javax.swing.ButtonGroup;
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JFileChooser;
+import javax.swing.JLabel;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.JScrollPane;
+import javax.swing.JSplitPane;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+import javax.swing.ScrollPaneConstants;
+import javax.swing.SwingConstants;
+import javax.swing.SwingUtilities;
+import javax.swing.border.TitledBorder;
+import javax.swing.filechooser.FileFilter;
+
+import edu.csus.ecs.pc2.VersionInfo;
+import edu.csus.ecs.pc2.core.ClipboardUtilities;
+import edu.csus.ecs.pc2.core.FileUtilities;
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.log.Log;
+import edu.csus.ecs.pc2.core.model.ClientSettings;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.model.Language;
+import edu.csus.ecs.pc2.core.model.Problem;
+import edu.csus.ecs.pc2.list.ListUtilities;
+import edu.csus.ecs.pc2.ui.team.QuickSubmitter;
+
+/**
+ * A UI that to submit files found in a CDP.
+ * 
+ * 
+ * @author Douglas A. Lane, PC^2 Team, pc2@ecs.csus.edu
+ */
+public class SubmitSampleRunsPane extends JPanePlugin {
+
+	/**
+	 * ClientSettings key for CDP Path
+	 */
+	private static final String CUSTOM_SUBMIT_SAMPLE_CDP_PATH = "CustomSubmitSampleCDPPath";
+
+	private static final long serialVersionUID = -8862440024499524533L;
+
+	private JTextField cdpTextField;
+
+	private JLabel messageLabel;
+
+	private JCheckBox checkBoxSubmitYesSamples;
+
+	private JCheckBox checkBoxSubmitFailingSamples;
+
+	private QuickSubmitter submitter = new QuickSubmitter();
+
+	private JTextArea textArea;
+
+	private SimpleDateFormat hhMMSSformatter = new SimpleDateFormat("hh:mm:ss");
+
+	private int linenumber = 0;
+	/**
+	 * @wbp.nonvisual location=162,159
+	 */
```

Comment:
What in the world does this comment have to do with anything?  (Is "wbp" a reference to William B. Poucher???)  If this comment has real meaning, please explain it.  If not, please delete it.